README.md

```diff
@@ -38,7 +38,7 @@ yarn install                           # Install project dependencies
 * go to https://www.strava.com/settings/api
 * retrieve application ID and secret
 * create a new .env file at the root of this project
-* set CLIENT_ID and CLIENT_SECRET keys with retrieved values.
+* edit .env file and add  CLIENT_ID and CLIENT_SECRET keys with your application ID and secret.
```

Comment:
paste the following code inside your .env file and make sure to replace [XXX] and [YYY] with your info
```
CLIENT_ID [XXX]
CLIENT_SECRET :[YYY]
```
or may be it is

```
CLIENT_ID=[XXX]
CLIENT_SECRET =[YYY]
```